src/Components/shared/Button.js

```diff
@@ -15,17 +18,38 @@ const styles = StyleSheet.create({
   button: {
     left: '10%',
     right: '10%',
-    width: '80%',
+    width: '90%',
     height: 40,
-    borderRadius: 30,
+    borderRadius: 5,
     backgroundColor: theme.COLOR_PRIMARY,
     padding: 10,
+    justifyContent: 'center',
+    alignItems: 'center',
   },
   text: {
     textAlign: 'center',
     color: theme.COLOR_PRIMARY_LIGHT,
     fontSize: theme.FONT_SIZE_MEDIUM,
   },
+  buttonLight: {
+    left: '10%',
```

Comment:
Prop-tip, you can use array-of-styles to avoid repetition in the styles. 
This is not a problem for this component, as it is pretty simple, but styles often get big, so avoiding repetition is good.

```jsx
const Button = props => {
  const btnColor = props.buttonLight ? styles.buttonLight : styles.buttonDark;

  return (
    <TouchableOpacity style={[styles.button, btnColor]} onPress={props.onPress} testID="button">
      {/* ...*/}
    </TouchableOpacity>
  );
};

const styles = StyleSheet.create({
  button: {
    left: '10%',
    right: '10%',
    /*... everything that is common */
  },
  buttonLight: {
    backgroundColor: theme.COLOR_PRIMARY,
  },
  buttonDark: {
    backgroundColor: theme.COLOR_PRIMARY_LIGHT,
    borderColor: theme.COLOR_SECONDARY,
  }
  /*...*/
});
```